### PR TITLE
Team Library UI

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

https://user-images.githubusercontent.com/99246719/212893069-2706ef88-f08e-428f-88f7-0f019c0e3aca.mov

View-only at the moment, basic navigation implemented.

## Related Issue(s)

https://github.com/flowforge/flowforge/issues/1368

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [-] Suitable unit/system level tests have been added and they pass
 - [x] Documentation has been updated
    - [-] Upgrade instructions
    - [-] Configuration details
    - [-] Concepts

### CODE REVIEW

**Comment by joepavitt** [comment]
```
<p>You can read more about <a href="https://nodered.org/docs/user-guide/editor/workspace/import-export" target="_blank">Import & Exporting Flows</a> and <a>Importing & Exporting Functions</a> in the Node-RED documentation</p>
```

Still looking for a link to the documentation of Node-RED that explains how to import/export functions in Node-RED.

**Comment by joepavitt** [comment]
> Still looking for a link to the documentation of Node-RED that explains how to import/export functions in Node-RED.

Seems no such documentation exists, so will remove the reference i the dialog

**Comment by joepavitt** [comment]
Not currently able to test the billable features in Cypress

**Comment by knolleary** [comment]
Having played with it, the fact the 'files' get a mouse pointer when you hover over them, but they don't do anything on click feels off. Do we have a way to keep the normal cursor for non-clickable entries?

We need to capture some issues around future enhancements to this view. I don't think there is a lot needed, but one more iteration would help it feel a bit more complete.
 - Show a preview of entries on click. Something we pushed out of scope for this iteration.
    We have no way to show a rendered view of the flow, but providing access to the JSON would also be valuable to allow a user to copy an entry without having to open up a NR editor.
 - Add 'delete' option (will require backend api as well)
 - Add an 'updated-at' column - that info is not currently in the meta data returned by the api but can easily be added.

**Comment by joepavitt** [comment]
> Having played with it, the fact the 'files' get a mouse pointer when you hover over them, but they don't do anything on click feels off. Do we have a way to keep the normal cursor for non-clickable entries?

That should be do-able

**Comment by joepavitt** [comment]
> We need to capture some issues around future enhancements to this view. I don't think there is a lot needed, but one more iteration would help it feel a bit more complete.

Agreed, will record once this is finalised.

File: frontend/src/pages/team/Library.vue [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        })
+    },
+    methods: {
+        loadLibrary (parentDir) {
+            return teamApi.getTeamLibrary(this.team.id, parentDir).then((library) => {
+                return library
+            })
+        },
+        entrySelected (entry) {
+            console.log(entry)
```
Comment (knolleary) [inline frontend/src/pages/team/Library.vue:65]:
```suggestion
```

**Review by knolleary — APPROVED** [review]
Removed one stray console.log, but otherwise, good for first iteration.

**Pull request merged**